source/_components/zabbix.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Zabbix"
+description: "Instructions how to integrate Zabbix into Home Assistant."
+date: 2016-12-13 22:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: g
+ha_category: 
+featured: false
+ha_release: 0.35
```

Comment:
Will need to be updated before being merged. Can wait until parent PR is merged to get the correct release number.